homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -300,6 +300,7 @@ async def _try_command(self, mask_error, func, *args, **kwargs):
         """Call a vacuum command handling error messages."""
         try:
             await self.hass.async_add_executor_job(partial(func, *args, **kwargs))
+            await self.hass.async_run_job(self.coordinator.async_refresh)
```

Comment:
Is there a reason to use `hass.async_run_job`? Normally we just await coroutines from inside coroutine functions.